File: sqlitejava/src/main/java/com/codekul/sqlitejava/AppDb.java
package com.codekul.sqlitejava;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;
import android.content.Context;


/**
 * Created by aniruddha on 16/11/17.
 */

@Database(entities = {Car.class}, version = 1)
public abstract class AppDb extends RoomDatabase{

    public abstract CarDao carDao();
}

File: sqlite/src/main/java/com/codekul/sqlite/db/AppDb.kt
package com.codekul.sqlite.db

import android.arch.persistence.room.Database
import android.arch.persistence.room.RoomDatabase
import com.codekul.sqlite.dao.CarDao
import com.codekul.sqlite.model.Car

/**
 * Created by aniruddha on 14/11/17.
 */

@Database(
        entities = arrayOf(
                Car::class
        ),
        version = 1
)
abstract class AppDb : RoomDatabase() {
    abstract fun carDao(): CarDao
}